k8s/externalsnapshotter/volumesnapshotclasses.go

```diff
@@ -0,0 +1,60 @@
+package externalsnapshotter
+
+import (
+	"github.com/kubernetes-csi/external-snapshotter/v2/pkg/apis/volumesnapshot/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// SnapshotClassOps is an interface to perform k8s VolumeSnapshotClass operations
+type SnapshotClassOps interface {
+	// CreateSnapshotClass creates the given snapshot
```

Comment:
nit: creates the given snapshot class
(same for all the comments below)

Reply:
Updated.